parsec/core/fs/workspacefs/workspacefs.py

```diff
@@ -4,16 +4,32 @@
 from collections import defaultdict
 from contextlib import asynccontextmanager
 from pathlib import Path
-from typing import TYPE_CHECKING, AsyncIterator, Awaitable, Callable, Dict, List, Tuple, cast
+from typing import TYPE_CHECKING, AsyncIterator, Awaitable, Callable, cast
 
 import attr
 import structlog
 import trio
 
-from parsec._parsec import CoreEvent, DateTime, FileManifest, RealmStatusRepOk, Regex
-from parsec.api.data import AnyRemoteManifest, BlockAccess
-from parsec.api.data import FileManifest as RemoteFileManifest
-from parsec.api.protocol import MaintenanceType, RealmID, UserID
+from parsec._parsec import (
+    BlockAccess,
+    CoreEvent,
+    DateTime,
+    EntryID,
+    EntryName,
+    FileManifest,
+    LocalDevice,
+    LocalFileManifest,
+    LocalFolderManifest,
+    LocalWorkspaceManifest,
+    MaintenanceType,
+    RealmID,
+    RealmStatusRepOk,
+    Regex,
+    UserID,
+    WorkspaceEntry,
+)
+from parsec._parsec import FileManifest as RemoteFileManifest
```

Comment:
Dupplicate. I think you can use the alias in the import tuple, eg
```
from parsec._parsec import (
    EntryID,
    EntryName,
    ...
    FileManifest as RemoteFileManifest
)
```

Reply:
Ruff force to have a new line when we write an alias :/